src/components/ha-gauge.ts

```diff
@@ -117,6 +166,15 @@ export class Gauge extends LitElement {
         transform-origin: 50% 100%;
         transition: all 1s ease 0s;
       }
+      .dial-hand {
+        fill: var(--gauge-color);
```

Comment:
```suggestion
        fill: var(--primary-color);
```

Reply:
Changed the needle design. It now uses `--primary-text-color`.